### PR TITLE
Loadout Update

### PR DESCRIPTION
**Pull request by Marrone321**
General Description: This PR updates several loadouts for followers, wastelanders, far-landers, and the Redwater faction.

FOLLOWERS CHANGES
image

STARTING TEXT
- Starting text, including Description, Enforces and Forbids, have been edited to reflect the standard the server wants to see and has also omitted references to NCR and the Legion.
- For Admin and Guard there is slightly different text to match their job descriptions.

LOADOUTS

- Removed CHEMWIZ trait which somehow fixed them not having CHEMWHIZ
- Professors now have Loadouts, two robust options which provide new machine boards to the Followers when they join as part of their loadout. The two loadouts are Environmental Scientist who has hydroponic boards, and Medical Specialist who has a blood bank.

- Specialists have some tweaks to make Medical Researcher more robust by adding a Bluespace syringe and an advanced health analyzer to their loadout.

- Randomly it seems the Volunteer loadout which has tools and construction stuff, had a chemist PDA. This has been removed and now the weakest loadout, the Student, has a PDA.

- Followers Guard had a tweak, the long range loadout has a scope, and the shotgun for the short range loadout has been changed to a police shotgun which is more inline with the aesthetic, and starts with bean bag rounds - though its total capacity is 6 as opposed to the previous 4.

These changes are intended to bring more value and encourage more players to participate in these roles. If you have any suggestions for additions, changes or subtractions let me know in the comments.

WASTELANDER CHANGES
image

LOADOUTS

Several additions and tweaks have been done to the Wastelander loadouts in order to properly reflect a myriad of playstyles.

- Small changes include, changing a welding helmet to welding goggles, adding extra magazines where there was only 1, tweaking the settler loadout to be more settler-y by giving them glass instead of wood, giving them a more melee focused build that resemble tools, and adding seeds to their loadout.  The Wasteland Medic has been tuned down with salts and surgery bag removed, and the Vaultie has lost their headset radio.

- 10 new Wastelander loadouts have been added, the Gambler which has a lot of interesting RP items, the Vaquero which allows players to explore another aesthetic in-line with the South West, the Hobo loadout for those who want a challenge, the Hombre loadout to replace the desert ranger one so it is more in-line with our current lore and to get away from New Vegas, Ex-Military for those who want to LARP as a soldier or mercenary, a brand new Brotherhood of Steel waster loadout that does not have grenades and is more balanced with other waster loadouts, Eidelon loadout for those who wish to be sneaky and slightly Russian if they so wish, the Aviator loadout to allow players some options to have that air pilot aesthetic, the Trapper loadout for that CLASSIC CLASSIC Fallout experience, and finally the Trouper loadout for all of the bottoms out there.

FAR-LANDER CHANGES
image
image

NEW TRIBAL TRADITIONS CRAFTING TRAITS AND GRANTER

- I have created a whole new set of traits and it took a lot of work, having to do multiple things seven times over and over to make a book that allows you to pick from a list a traitbook which makes it so you can craft one of the seven different former tribes armor and garments. Rejoice!

LOADOUTS 

- The loadouts for Far-Landers have been reduced from 21-3 to 5. To those who wish for aesthetic or loadout options which have been omitted from this decision, let me know and I can tweak some things or add another class since I removed a lot, but they must be more generalized so that specific tribes arent tied to actual loadout options.

REDWATER CHANGES

image

LOADOUTS

- Redwater Slave, my favourite job, no longer has explosive collars; they are now shock collars. Aww shucks, I wanted to be round ended!

- A whole new job was added called Redwater Resident. They will be in charge of supervising and protecting Redwater, and act as inhabitants of the town; they may travel outside of the town to gather materials but otherwise should be staying in the area and around town. They are equipped in quite a robust manner, so anyone who dares to battle the town better be geared to the teeth.

- The Overboss keeps spawning in Nash naked. I fixed this. They now have clothes, and also spawn in Redwater.

## Pre-Merge Checklist
- [X] You tested this on a local server. (ALL CHANGES HAVE BEEN TESTED... EXCEPT THE SHOCK COLLAR. THAT IS IN GODS HANDS.)
- [X] This code did not runtime during testing. (NO RUNTIME OR EXCESS LAG/ISSUES OCCURED DURING TESTING)
- [X ] You documented all of your changes. (YEEEE BOY)

## Changelog
:cl:
add: Updated and Added Follower of Apocolypse, Wastelander, and Redwater Resident Loadouts
add: Added an entirely new trait system for Far-Landers
del: A ton of excessive loadouts from Far-Landers
tweak: A variety of loadouts so that they are consistent with eachother and do not have so much outward advantage when weighed with another, to the best of my ability.
balance: Balanced loadouts and the weight of factions by adding a New Job to Redwater so that it has more population and more people RPing over there.
fix: They really done made the Overboss spawn in Nash NAKED. We fixed that.
/:cl:

### CODE REVIEW

**Review by Tk420634 — APPROVED** [review]
Marry, you're an unironic angel.  Thank you so much. <3

**Comment by Tasald** [comment]
Are you sure the professor and admin can still use the machines? I had to manually add all the recipies to the loadouts to fix the issue in a previous pr.l but with out it they might not beable to use the auto doc or chem master.

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -339,6 +309,252 @@ Raider
...
+	belt = /obj/item/storage/belt/military/assault
+	shoes = /obj/item/clothing/shoes/laceup
+	uniform = /obj/item/clothing/under/f13/densuit
+	suit = /obj/item/clothing/suit/armor/medium/vest
+	gloves =  /obj/item/clothing/gloves/color/white
+	head = /obj/item/clothing/head/fedora
+	neck = /obj/item/storage/belt/holster/ranger44
+	mask =  /obj/item/clothing/mask/bandana/durathread
+	backpack_contents = list(
+		/obj/item/gun/ballistic/automatic/smg/greasegun/mid = 1,
```
Comment (Superlagg) [inline code/modules/jobs/job_types/wasteland.dm:447]: I totally forgot these were a thing, I havent rebalanced the different crafting tiers of SMGs. Luckily this one's pretty much fine, it doesnt have any vars on it, but most of the others have funky damage multipliers. So, if anyone complains a gun does like 10 times the damage, that's why. id replace it with just a greasegun cus, really, theyre all the same.

Comment (Superlagg) [reply code/modules/jobs/job_types/wasteland.dm:447]: this is a me-problem btw

**Review by Superlagg — APPROVED** [review]

**Comment by Marrone321** [comment]
> Are you sure the professor and admin can still use the machines? I had to manually add all the recipies to the loadouts to fix the issue in a previous pr.l but with out it they might not beable to use the auto doc or chem master.

I will go test that now

File: code/modules/jobs/job_types/wasteland.dm (hunk 1 above)
Comment (Marrone321) [reply code/modules/jobs/job_types/wasteland.dm:447]: Yeah I can easily fix that

**Comment by Marrone321** [comment]
> Are you sure the professor and admin can still use the machines? I had to manually add all the recipies to the loadouts to fix the issue in a previous pr.l but with out it they might not beable to use the auto doc or chem master.

Yeah professor and admin can indeed use chem master and dispenser and autodoc now

**Pull request merged**